index.html

```diff
@@ -0,0 +1,188 @@
+                    <ul class="mobile-menu-ul">
+                        <li  class="mobile-menu-option"><a href="#portfolio">Portfolio</a></li>
+                        <li  class="mobile-menu-option"><a href="#about">About</a> </li> 
+                        <li  class="mobile-menu-option"><a href="#contact">Contact</a> </li>
+                       </ul>
+                </div>
+            </div>
+            <div class="dash"></div>
+    </section>
+        <section class="second">
+          <h1> Hey there. I’m Alex.<span class="green" > <br> I’m a software developer</span></h1>
+        <p class="second-text">
+        I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.</p>
+            <ul class ="socials">
+                <li class="socials-list"><a href="https://twitter.com/BadruNdawula"><img src="./images/twitter.jpg" alt=""class="socials-list"></a></li>
+                <li class="socials-list"><a href="https://www.linkedin.com/in/badru-ndawula-32b371236/"><img src="./images/Linkedin icon.jpg" alt=""class="socials-list"></a></li>
+                <li class="socials-list"><a href="https://github.com/PrinceBadru"><img src="./images/github.jpg" alt="" class="socials-list"></a></li>
+                <li class="socials-list"><a href="#"><img src="./images/medium.jpg" alt="" class="socials-list"></a></li>
+                <li class="socials-list"><a href="#"><img src="./images/angellist.jpg" alt="" class="socials-list"></a></li>
+            </ul> <br>
+    
+            <div class="dis"><img src="./images/Disabled.png" alt="" class="disabled"> </div>
+        </section>
+               
+               
+        <section class = "work-section">
+            <h2 class ="recent" >My Recent Works</h2>
+            <hr class="line">
+           
+        </section>
+    
+                <section class="aboutme-background" id="about">
+                <section class="aboutme section">
+                    <div class="aboutme-container">
+                    <div class="about2">
+                        <h2>About me</h2>
+    
+                    <div class="about">
+                    </div>
+                    <div class="about-text">
+                    I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.
+                    </div>
+                    <div class="butt resume-container"><button class="btn btn-resume" type="button"> Get My Resume</button></div>
+                </div>
+                    </div>  
+                    
+                    <div class="skills-container">
+                    <article class="project2">
```

Comment:
- [ ] Please note that we don't need to use the (`article` tag) here. The `<article>` HTML element is used to represent a self-contained composition in an HTML document, such as a blog post, a forum post, or a news story. But that isn't what we want to accomplish here so kindly change all the article tags in your document to a (`div` tags). :100: 
- [ ] Please make use of `display : flex` to align the contents of each `project2` container. This makes it easy to create a flexible, responsive, and adaptable layout. It will also allow you to control the size and position of elements within each container :wink: